### DISCUSSION TITLE
Our project README.

### DISCUSSION THREAD

**Discussion by linahKhayri** [MIT-Emerging-Talent/ET6-foundations-group-24#18] [Announcements]
## 👋 Hello Guys,
I didn't know about the discussions feature here before but it seems cool! Let’s use it to chat here. 

I created a new branch called repo-readme to start making some edits to the project's README.
I’ve already started adding some changes so take a look and let me know what you think and what we should add or delete.
[here's the link](https://github.com/MIT-Emerging-Talent/ET6-foundations-group-24/blob/repo-readme/README.md)

Also, when I wrote our names, I've put them in alphabetical order so please make sure that I didn't make any mistakes with anyone's name.

**Comment by Khusro-S** [comment]
A job well done 🫡. Just a small idea. Just like we have links to the communication, group norms and stuff, how about we make the dream team members' names links to their GitHub profiles too?

**Reply by Khusro-S** [reply]
And, if I remember correctly, we still need to come up with a team name. Here are some suggestions I thought of:

- The Keyboard Warriors
- Bug Busters
- The Error Finders
- Code Sprouts
- The Code Explorers

**Reply by Zeinab15** [reply]
@Khusro-S  I like Bug Busters

**Reply by linahKhayri** [reply]
> A job well done 🫡. Just a small idea. Just like we have links to the communication, group norms and stuff, how about we make the dream team members' names links to their GitHub profiles too?

omg yess, I've been thinking about making these names clickable links too, that's a good idea thank you Khusro!!!

**Reply by linahKhayri** [reply]
> And, if I remember correctly, we still need to come up with a team name. Here are some suggestions I thought of:
> 
> * The Keyboard Warriors
> * Bug Busters
> * The Error Finders
> * Code Sprouts
> * The Code Explorers

I LOVE IT
